geoportailv3/static/js/layermanager/layermanagerdirective.js

```diff
@@ -0,0 +1,47 @@
+goog.provide('app.layermanagerDirective');
+
+goog.require('app');
+
+
+/**
+ * @param {string} appLayermanagerTemplateUrl Url to layermanager template
+ * @return {angular.Directive} The Directive Definition Object.
+ */
+app.layermanagerDirective = function(appLayermanagerTemplateUrl) {
+  return {
+    restrict: 'E',
+    scope: {
+      'map': '=appLayermanagerMap',
+      'layers': '=appLayermanagerLayers'
+    },
+    controller: 'AppLayermanagerController',
+    controllerAs: 'layermanagerCtrl',
```

Comment:
I don't think we need "complex" name for directive controllers. The name is local to the directive, so I think that we can just use "ctrl".